examples/expositional/models/snowflake_cortex/arctic_quickstart.ipynb

```diff
@@ -188,41 +187,15 @@
     "        \"\"\"\n",
     "        Generate answer from context.\n",
     "        \"\"\"\n",
-    "\n",
-    "        def escape_string_for_sql(input_string):\n",
-    "            escaped_string = input_string.replace(\"\\\\\", \"\\\\\\\\\")\n",
-    "            escaped_string = escaped_string.replace(\"'\", \"''\")\n",
-    "            return escaped_string\n",
-    "\n",
-    "        prompt = escape_string_for_sql(f\"\"\"\n",
+    "        prompt = f\"\"\"\n",
     "         We have provided context information below. \n",
     "            {context_str}\n",
     "            Given this information, please answer the question: {query}\n",
-    "        \"\"\")\n",
-    "\n",
-    "        cursor = snowflake_session.connection.cursor()\n",
-    "        try:\n",
-    "            # We use `snowflake.connector.cursor.SnowflakeCursor::execute` to\n",
-    "            # execute the query instead of\n",
-    "            # `snowflake.snowpark.session.Session::sql` since the latter is not\n",
-    "            # thread-safe.\n",
-    "            res = cursor.execute(f\"\"\"SELECT SNOWFLAKE.CORTEX.COMPLETE(\n",
-    "                'snowflake-arctic',\n",
-    "                [\n",
-    "                {{'role': 'user', 'content': '{prompt}'}}\n",
-    "                ], {{\n",
-    "                    'temperature': 0\n",
-    "                }}\n",
-    "                )\"\"\").fetchall()\n",
-    "        finally:\n",
-    "            cursor.close()\n",
-    "\n",
-    "        if len(res) == 0:\n",
-    "            return \"No response from cortex function\"\n",
-    "        completion = json.loads(res[0][0])[\"choices\"][0][\"messages\"]\n",
-    "        print(\"full response from cortex function:\")\n",
-    "        print(res)\n",
-    "        return completion\n",
+    "        \"\"\"\n",
+    "        resp = Complete(model='snowflake-arctic', prompt=[{'role': 'user', 'content': prompt}], session=snowflake_session)\n",
```

Comment:
The latest `Complete` (hence the version check >= 1.7.1 of `snowflake-ml-python`) makes decision of SQL vs REST / corvo backend based on the type of arguments. 

It will hit the sql function iff `prompt` is of  type `snowpark.Column`